installscripts/jazz-terraform-unix-noinstances/operationalUser.tf

```diff
@@ -0,0 +1,70 @@
+resource "aws_iam_user" "operational" {
+  name = "${var.envPrefix}-operationaluser"
+  path = "/system/"
+}
+
+resource "aws_iam_access_key" "operational_key" {
+  user = "${aws_iam_user.operational.name}"
+}
+
+data "aws_caller_identity" "current" {}
+
+resource "aws_iam_user_policy" "operational_policy" {
+  name = "${var.envPrefix}-operationaluser-policy"
+  user = "${aws_iam_user.operational.name}"
+
+  policy = <<EOF
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Action": [
+        "iam:PassRole",
+        "apigateway:*",
+        "lambda:*",
+        "logs:*",
+        "cloudfront:*",
+        "cloudformation:ValidateTemplate"
+      ],
+      "Effect": "Allow",
+      "Resource": "*"
+    },
+    {
+      "Action": [
+        "cloudformation:*"
+      ],
+      "Effect": "Allow",
+      "Resource": [
+          "arn:aws:cloudformation:*:${data.aws_caller_identity.current.account_id}:stackset/${var.envPrefix}*:*",
```

Comment:
Ok, so the operational user will be restricted to resources that only the installing user would have access to in the first place.

Do we need that restriction/will that restriction cause complications or gotchas later, or should we just restrict based on `envPrefix`?

Reply:
Yeah, each resource have their own pattern to specify resource based on envPrefix.

Reply:
@bleggett Since Jazz supports AWS Events ie DynamoDB, Kinesis, S3, SQS, Users can use anyname other than envPrefix.